python/paddle/fluid/contrib/slim/tests/quantization/config_mkldnn_int8.yaml

```diff
@@ -4,6 +4,7 @@ strategies:
         class: 'MKLDNNPostTrainingQuantStrategy'
         int8_model_save_path: 'OUTPUT_PATH'
         fp32_model_path: 'MODEL_PATH'
+        omp_num_threads: 4
```

Comment:
How about `cpu_math_library_num_threads`?

Reply:
sounds good, I will change the name.

Reply:
done